### PR TITLE
RHESSI Summary Light Curve Fix

### PR DESCRIPTION
**Pull request by ehsteve**
This fixes a bug whereby the count rates were incorrectly interpreted from the observing summary files. They are stored as a single byte which must be converted to an actual count rate.

### CODE REVIEW

File: sunpy/instr/rhessi.py [hunk 1]
```diff
@@ -264,17 +264,57 @@ def parse_obssumm_file(filename):
...
+        A compressed count rate returned from an observing summary file.
+
+    References
+    ----------
+    Hsi_obs_summ_decompress.pro `<http://hesperia.gsfc.nasa.gov/ssw/hessi/idl/qlook_archive/hsi_obs_summ_decompress.pro>`_
+    """
+    ll = np.arange(0, 16, 1)
+    lkup = np.zeros(256, dtype='int')
+    sum = 0
+    for i in np.arange(0, 16):
```
Comment (dpshelio) [inline sunpy/instr/rhessi.py]: why using `np.arange` for indices? would not just `range` would do?

**Comment by dpshelio** [comment]
If it's a bug fix, should it be also pushed to 0.6?

**Comment by Cadair** [comment]
@dpshelio yeah this should be back ported.

**Comment by ayshih** [comment]
:+1: when tests pass

**Comment by Cadair** [comment]
:+1: when travis says ok

**Comment by ehsteve** [comment]
rebased!

**Comment by Cadair** [comment]
@ehsteve this is fine as is.

For the 0.6 branch you will need to do this:

```
git fetch upstream
git checkout -b rhessi_lc_fix_06
git rebase -i upstream/0.6

<fix commits list>

git push origin rhessi_lc_fix_06

<open new PR>
```

**Comment by ehsteve** [comment]
@Cadair! merge me!

**Comment by Cadair** [comment]
@ehsteve backport please!

**Comment by ehsteve** [comment]
@Cadair done.

**Pull request merged**